VERSION

```diff
@@ -1 +1 @@
-1.11.0-DEV
+1.11.0-DEV
```

Comment:
Looks like you deleted a newline?  You'll want to make it so that this file is not changed at all.

Reply:
Sorry, indeed. Shouldn't have hurried through this one. Now this is resolved.